src/main/java/aquality/appium/mobile/elements/Element.java

```diff
@@ -86,4 +88,9 @@ public void sendKeys(Keys key) {
         logElementAction("loc.text.sending.keys", Keys.class.getSimpleName().concat(".").concat(key.name()));
         doWithRetry(() -> getElement().sendKeys(key));
     }
+
+    @Override
+    public IElementTouchActions getTouchActions() {
+        return new ElementTouchActions(this);
```

Comment:
Would you like to bind it in AqualityServices and get this like AqualityServices.get<IElementTouchActions>

Reply:
As far as I'm concerned,  `ElementTouchActions` has at least 1 argument, that's why we won't be able to create an injector.